020-Connect/020-JavaScript/020-prisma.mdx

```diff
@@ -75,12 +77,12 @@ const result = await prisma.users.findMany({
 });
 ```
 
-## Shadow Database
+## Migrations
 
-Migrations with Prisma require a [shadow database](https://www.prisma.io/docs/orm/prisma-migrate/understanding-prisma-migrate/shadow-database).
+Migrations with Prisma require a [Shadow Database](https://www.prisma.io/docs/orm/prisma-migrate/understanding-prisma-migrate/shadow-database), which is an extra database that stores migration information and state.
 
-For this purpose, create an additional database in Xata and configure its Postgres endpoint as `shadowDatabaseUrl` in the datasource, as suggested by the relevant [Prisma documentation](https://www.prisma.io/docs/orm/prisma-migrate/understanding-prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually).
+Create an additional database in Xata (we recommend naming it databasename-migrations), configure this database's [Postgres connection string](/docs/postgres#connection-string) as the `shadowDatabaseUrl` in the Prisma datasource and store the connection string in your `.env` file as `SHADOW_DATABASE_URL`. Refer to Prisma's [documentation](https://www.prisma.io/docs/orm/prisma-migrate/understanding-prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually) for more details.
```

Comment:
We could also add the shadowUrl in the code sample for the `schema.prisma` file, a bit above, so:

```
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
  shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
}
```

I think it makes sense to always recommend this, even if technically only needed if you do migrations.

Reply:
Thanks for the suggestion, added it